### PR TITLE
[fix/request-transformer] concatenating value to header/querystring

### PR DESCRIPTION
**Pull request by shashiranjan84**
Part of the fix for issue #393. New property `concat` added to support adding new value to existing header/querystring. If header/querystring does not exist, a new one will be added

### CODE REVIEW

File: kong/plugins/request-transformer/access.lua [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
+local req_get_headers = ngx.req.get_headers
+local req_read_body = ngx.req.read_body
+local req_set_body_data = ngx.req.set_body_data
+local req_get_body_data = ngx.req.get_body_data
+local req_clear_header = ngx.req.clear_header
+local req_get_post_args = ngx.req.get_post_args
+local encode_args = ngx.encode_args
+local type = type
+local table_size = utils.table_size
+local stringy_strip = stringy.strip
```
Comment (thibaultCha) [inline kong/plugins/request-transformer/access.lua]: that is not necessary, stringy is already a local.

File: kong/plugins/request-transformer/access.lua [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+local req_set_header = ngx.req.set_header
+local req_get_headers = ngx.req.get_headers
+local req_read_body = ngx.req.read_body
+local req_set_body_data = ngx.req.set_body_data
+local req_get_body_data = ngx.req.get_body_data
+local req_clear_header = ngx.req.clear_header
+local req_get_post_args = ngx.req.get_post_args
+local encode_args = ngx.encode_args
+local type = type
+local table_size = utils.table_size
```
Comment (thibaultCha) [inline kong/plugins/request-transformer/access.lua]: same

Comment (shashiranjan84) [inline kong/plugins/request-transformer/access.lua]: fixed

**Comment by thibaultCha** [comment]
Same comment as #728

**Comment by thibaultCha** [comment]
up too, like #728

**Comment by shashiranjan84** [comment]
PR can be reviewed for merge.

**Comment by shashiranjan84** [comment]
closing and creating new PR against next

**Pull request closed without merging**